quinn-proto/src/connection/mod.rs

```diff
@@ -77,6 +77,23 @@ where
     /// Exactly one prior to `self.rem_cids.offset` except during processing of certain
     /// NEW_CONNECTION_ID frames.
     rem_cid_seq: u64,
+    rem_cid_resettoken: Option<ResetToken>,
```

Comment:
We're currently storing the current reset token in `self.peer_params.stateless_reset_token`. Is this redundant to that?

Reply:
Good catch. I noticed before that `self.peer_params.stateless_reset_token` has reset token but i chose to add another field anyway. Will revisit my previous decision and see if i can get rid of it